### DISCUSSION TITLE
v0.13.2

### DISCUSSION THREAD

**Discussion by ErikBjare** [ActivityWatch/activitywatch#1102] [Releases]
These are the release notes for ActivityWatch version v0.13.2.

**New to ActivityWatch?** Check out the [website](https://activitywatch.net) and the [README](https://github.com/ActivityWatch/activitywatch/blob/master/README.md).

# Installation

See the [getting started guide in the documentation](https://docs.activitywatch.net/en/latest/getting-started.html).

# Downloads

 - [**Windows**](https://github.com/ActivityWatch/activitywatch/releases/download/v0.13.2/activitywatch-v0.13.2-windows-x86_64-setup.exe) (.exe, installer)
 - [**macOS**](https://github.com/ActivityWatch/activitywatch/releases/download/v0.13.2/activitywatch-v0.13.2-macos-x86_64.dmg) (.dmg)
 - [**Linux**](https://github.com/ActivityWatch/activitywatch/releases/download/v0.13.2/activitywatch-v0.13.2-linux-x86_64.zip) (.zip)

# Contributors

Thanks to everyone who contributed to this release:

@2e3s, @BasileusErwin, @BelKed, @brayo-pip, @davidfraser, @erikbjare, @Shi-Soul

# Changelog

Changes since v0.13.1:

### 🔨 Misc (7)
<details><summary>Click to expand</summary>
<p>

 - chore: bump version to 0.13.2 ([`839d99f`](https://github.com/ActivityWatch/activitywatch/commit/839d99f))
 - chore: updated changelog contributors cache ([`78d2b9d`](https://github.com/ActivityWatch/activitywatch/commit/78d2b9d))
 - docs: fix mermaid architecture diagram ([`0951338`](https://github.com/ActivityWatch/activitywatch/commit/0951338))
 - docs: added links to the mermaid architecture diagram ([`cdc8d5e`](https://github.com/ActivityWatch/activitywatch/commit/cdc8d5e))
 - docs: added mermaid-diagram for visualization of architecture ([#1086](https://github.com/ActivityWatch/activitywatch/issues/1086)) ([`43cbad1`](https://github.com/ActivityWatch/activitywatch/commit/43cbad1))
 - chore: updated changelog_contributors.csv cache ([`26fe779`](https://github.com/ActivityWatch/activitywatch/commit/26fe779))
 - chore: bumped version to v0.13.1 in various places, including in submodules ([`6f0518a`](https://github.com/ActivityWatch/activitywatch/commit/6f0518a))

</p>
</details>

*(excluded 5 less relevant [commits](https://github.com/ActivityWatch/activitywatch/compare/v0.13.1...v0.13.2))*

### 🔨 Misc (1)

 - chore: bumped version to 0.13.1 ([`72108cd`](https://github.com/ActivityWatch/aw-server/commit/72108cd))

*(excluded 3 less relevant [commits](https://github.com/ActivityWatch/aw-server/compare/09f4e4c...b4ad075))*

### ✨ Features (3)

 - feat(ux): improve timeline usability ([#610](https://github.com/ActivityWatch/aw-webui/issues/610)) ([`291da6f`](https://github.com/ActivityWatch/aw-webui/commit/291da6f))
 - feat: revert to the default coloring for non-standard watchers ([#613](https://github.com/ActivityWatch/aw-webui/issues/613)) ([`85c007e`](https://github.com/ActivityWatch/aw-webui/commit/85c007e))
 - feat: use category colors in the timeline view ([#609](https://github.com/ActivityWatch/aw-webui/issues/609)) ([`0cf7831`](https://github.com/ActivityWatch/aw-webui/commit/0cf7831))

### 🐛 Fixes (3)

 - fix: correctly invert colors of alerts in dark mode ([#591](https://github.com/ActivityWatch/aw-webui/issues/591)) ([`e6ad4cd`](https://github.com/ActivityWatch/aw-webui/commit/e6ad4cd))
 - fix: add a default color for Media category ([#593](https://github.com/ActivityWatch/aw-webui/issues/593)) ([`6ed3e83`](https://github.com/ActivityWatch/aw-webui/commit/6ed3e83))
 - fix: add browser app name `Microsoft Edge Beta` on macOS ([#595](https://github.com/ActivityWatch/aw-webui/issues/595)) ([`ba1305d`](https://github.com/ActivityWatch/aw-webui/commit/ba1305d))

### 🐛 Fixes (2)

 - fix(sync): remove bug-causing workaround for spurious small db files (fixes [#466](https://github.com/ActivityWatch/aw-server-rust/issues/466)) ([#492](https://github.com/ActivityWatch/aw-server-rust/issues/492)) ([`5738246`](https://github.com/ActivityWatch/aw-server-rust/commit/5738246))
 - fix(linux): add sd-notify for better systemd interop ([#489](https://github.com/ActivityWatch/aw-server-rust/issues/489)) ([`8d7ce89`](https://github.com/ActivityWatch/aw-server-rust/commit/8d7ce89))

### 🔨 Misc (1)

 - chore: bumped aw-server version to 0.13.1 ([`b8330f1`](https://github.com/ActivityWatch/aw-server-rust/commit/b8330f1))

*(excluded 6 less relevant [commits](https://github.com/ActivityWatch/aw-server-rust/compare/3531ead...656f3c9))*

### ✨ Features (3)

 - feat(ux): improve timeline usability ([#610](https://github.com/ActivityWatch/aw-webui/issues/610)) ([`291da6f`](https://github.com/ActivityWatch/aw-webui/commit/291da6f))
 - feat: revert to the default coloring for non-standard watchers ([#613](https://github.com/ActivityWatch/aw-webui/issues/613)) ([`85c007e`](https://github.com/ActivityWatch/aw-webui/commit/85c007e))
 - feat: use category colors in the timeline view ([#609](https://github.com/ActivityWatch/aw-webui/issues/609)) ([`0cf7831`](https://github.com/ActivityWatch/aw-webui/commit/0cf7831))

### 🐛 Fixes (3)

 - fix: correctly invert colors of alerts in dark mode ([#591](https://github.com/ActivityWatch/aw-webui/issues/591)) ([`e6ad4cd`](https://github.com/ActivityWatch/aw-webui/commit/e6ad4cd))
 - fix: add a default color for Media category ([#593](https://github.com/ActivityWatch/aw-webui/issues/593)) ([`6ed3e83`](https://github.com/ActivityWatch/aw-webui/commit/6ed3e83))
 - fix: add browser app name `Microsoft Edge Beta` on macOS ([#595](https://github.com/ActivityWatch/aw-webui/issues/595)) ([`ba1305d`](https://github.com/ActivityWatch/aw-webui/commit/ba1305d))

### ✨ Features (1)

 - feat: add `exclude_titles` config & CLI option for excluding window titles by regex ([#99](https://github.com/ActivityWatch/aw-watcher-window/issues/99)) ([`76c85c6`](https://github.com/ActivityWatch/aw-watcher-window/commit/76c85c6))

### 🐛 Fixes (1)

 - fix: added exception handling for `Xlib.error.BadWindow` ([#97](https://github.com/ActivityWatch/aw-watcher-window/issues/97)) ([`7a89db4`](https://github.com/ActivityWatch/aw-watcher-window/commit/7a89db4))

### 🔨 Misc (1)

 - chore: bump version to 0.5.17 ([`6a58ec5`](https://github.com/ActivityWatch/aw-core/commit/6a58ec5))

### 🐛 Fixes (4)

 - fix: fixed lints and type issues ([`ece2b4c`](https://github.com/ActivityWatch/aw-client/commit/ece2b4c))
 - fix: fixed operator precedence bug in query ([#86](https://github.com/ActivityWatch/aw-client/issues/86)) ([`292b235`](https://github.com/ActivityWatch/aw-client/commit/292b235))
 - fix(example): fixed tz-naive datetimes in time_spent_today.py ([`ff61703`](https://github.com/ActivityWatch/aw-client/commit/ff61703))
 - fix(example): replace single quotes with double quotes in query ([#83](https://github.com/ActivityWatch/aw-client/issues/83)) ([`c699f43`](https://github.com/ActivityWatch/aw-client/commit/c699f43))

### 🔨 Misc (1)

 - format: formatted with black ([`38acb84`](https://github.com/ActivityWatch/aw-client/commit/38acb84))

*(excluded 2 less relevant [commits](https://github.com/ActivityWatch/aw-client/compare/5c49cb2...ece2b4c))*

**Full Changelog**: https://github.com/ActivityWatch/activitywatch/compare/v0.13.1...v0.13.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ActivityWatch/activitywatch/releases/tag/v0.13.2'>v0.13.2</a>.</em>

**Comment by Heiszicky** [comment]
app not running on windows but the localhost works perfectly on browser
